functionaltests/src/test/java/specs/response/HealthResponseFixture.java

```diff
@@ -0,0 +1,36 @@
+package specs.response;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.entity.ContentType;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.concordion.api.MultiValueResult;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.json.JSONObject;
+import org.junit.runner.RunWith;
+
+/**
+ * Simple /health URL response checker
+ */
+@RunWith(ConcordionRunner.class)
+public class HealthResponseFixture extends BasePaymentFixture {
+
+    public String getUrl() throws Exception {
+        return getHealthUrl();
+    }
+
+    public MultiValueResult handleHealthResponse() throws Exception {
+        final HttpResponse httpResponse = Request.Get(getHealthUrl())
+                .connectTimeout(200)
```

Comment:
well, IMHO for this exactly resource i think it should be like this value, since it does nothing except returning text. Guess, even in future this kind of time limit should stay there to avoid _heavy_ implementations of the resource.
I'd rather specify comment in the test why to keep this value really small than increase the timeout.